projects/ngrx.io/content/guide/migration/v13.md

```diff
@@ -109,8 +109,8 @@ When using Jasmine, reset the mock store after each test by adding the following
 ```ts
 import { MockStore } from '@ngrx/store/testing';
 
-jasmine.getEnv().afterEach(() => {
-  TestBed.inject(MockStore)?.resetSelectors();
+afterEach(() => {
+  getTestBed().inject(MockStore, null)?.resetSelectors();
```

Comment:
Is the change from `TestBed` to `getTestBed()` required?
If so, could you add an import statement please.

Reply:
@mangei pinging you so we can merge this one

Reply:
It is probably not required, see https://angular.io/guide/testing-utility-apis.  
But current code example use getTestBed().   
Also your platform uses it that way: https://github.com/ngrx/platform/blob/master/projects/ngrx.io/src/test.ts

I added the import.